tests/infra_test.go

```diff
@@ -219,12 +219,6 @@ var _ = Describe("Infrastructure", func() {
 		}, 300)
 
 		It("should include the network metrics for a running VM", func() {
-			By("Expecting the VirtualMachineInstance console")
-			expecter, err := tests.LoggedInAlpineExpecter(vmi)
-
-			Expect(err).ToNot(HaveOccurred())
-			defer expecter.Close()
-
```

Comment:
We added this expecter to act as sync point to make sure the VM completed the boot, and to reduce risk of false negatives. Do you think this risk is mitigated elsewhere? (or, even better, no longer relevant?)

Reply:
Ah I see, I didn't think about this reason. So now we have the expecter at the BeforeAll that runs the writing to the disk which renders this redundant I think, right?

Reply:
Added a comment to the relevant place

Reply:
A comment is good enough for me.